src/main/java/Storage.java

```diff
@@ -0,0 +1,64 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Storage {
+    private Path savePath;
+
+    Storage(Path savePath) {

```

Comment:
Access modifier missing here perhaps?

Reply:
What access modifier do you recommend I add?